src/app/components/MediaLoader/utils/buildPlaceholderConfig/index.ts

```diff
@@ -8,7 +8,7 @@ type BuildPlaceholderConfigParams = {
   title: string;
   duration: number;
   holdingImageURL: string;
-  placeholderImageOriginCode: string;
+  placeholderImageOriginCode?: string;
```

Comment:
Let's remove this as an optional parameter (we are now setting the value in our tvMedia code). We could perhaps use a type... e.g. 'pips'